day_6/exercises/burrito.js

```diff
@@ -16,6 +16,21 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping(newTopping){
+    this.toppings.push(newTopping)
+    console.log("add "+ newTopping);
+  }
+
+  removeTopping(lessTopping){
+    this.toppings.shift()
```

Comment:
This `lessTopping` argument is not the one actually being removed here. `Array.shift()` simply removes _the first item_ in the array.  Is there an alternative array method you could use here to remove a particular item?

Reply:
I could use .splice() method to remove an item from an array.

Reply:
Yep, great idea!